test/distributed/org/apache/cassandra/distributed/test/sai/StrictFilteringTest.java

```diff
@@ -55,11 +52,10 @@ public static void setUpCluster() throws IOException
     }
 
     @Test
-    public void shouldRejectNonStrictIN()
+    public void shouldPostFilterNonStrictIN()
     {
         CLUSTER.schemaChange(withKeyspace("CREATE TABLE %s.reject_in (k int PRIMARY KEY, a int, b int) WITH read_repair = 'NONE'"));
         CLUSTER.schemaChange(withKeyspace("CREATE INDEX ON %s.reject_in(a) USING 'sai'"));
-        CLUSTER.schemaChange(withKeyspace("CREATE INDEX ON %s.reject_in(b) USING 'sai'"));
```

Comment:
It passes and hits the same path even if we do not remove the second index

Reply:
Right. I just wanted to avoid creating the index if we didn't have to.